src/components/Card.js

```diff
@@ -21,15 +43,47 @@ export default class Card {
         })
     }
 
-    _handleLikeClick (){
-        this._likeButton.classList.toggle('active');
+    escPress(evt) {
```

Comment:
Please, delete it

Every modal should be created using `Popup`. And submit handler should be added inside the class